internal/dispatch/remote/cluster_test.go

```diff
@@ -0,0 +1,99 @@
+package remote
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/test/bufconn"
+
+	"github.com/authzed/spicedb/internal/dispatch/keys"
+	core "github.com/authzed/spicedb/pkg/proto/core/v1"
+	v1 "github.com/authzed/spicedb/pkg/proto/dispatch/v1"
+)
+
+type fakeDispatchSvc struct {
+	v1.UnimplementedDispatchServiceServer
+}
+
+func (fds *fakeDispatchSvc) DispatchCheck(context.Context, *v1.DispatchCheckRequest) (*v1.DispatchCheckResponse, error) {
+	// NOTE: must be larger than the configured timeout below for testing.
+	time.Sleep(20 * time.Millisecond)
```

Comment:
Could make this value a constant and then panic in the test case if the table entry has a timeout greater than it.

When possible we should strive to use code to enforce invariants rather than comments.